src/server/enrollment-system.js

```diff
@@ -1039,7 +1039,12 @@ function veteranToDemographicsInfo(veteran) {
           addressTypeCode: 'P',  // TODO(awong): this code is from VHA Standard Data Service (ADRDEV01) Address Type List P==Permanent. Determine if we need it.
         },
         emails: {
-          email: veteran.email,
+          email: [
+            {
+              address: veteran.email,
+              type: '1'
```

Comment:
I've been going off of a VOA Data Elements excel doc and the comments above this method - is there a a better place to look?